src/contexts/alert_handler.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { createContext, useContext, useState } from "react";
+
+const AlertHandlerContext = createContext();
+
+export const AlertHandlerProvider = ({ children }) => {
+  const [alertState, setAlertState] = useState({
+    visible: false,
+    errorText: "",
+  });
+
+  const setAlert = (errorText) => {
+    // TODO: ここを埋める
+    setAlertState({ visible: true, errorText: errorText });
+  };
+
+  const closeAlert = () => {
+    // TODO: ここを埋める
+    setAlertState({ visible: false });
+  };
```

Comment:
```suggestion
  const setAlert = (errorText) => {
    setAlertState({ visible: true, errorText: errorText });
  };

  const closeAlert = () => {
    setAlertState({ visible: false });
  };
```
TODO コメントが残っていると、対応が必要なのかどうかがわからず困ってしまいます。
対応完了した部分については、必ずコメントを削除するようにしてください！